### ISSUE TITLE
`block-no-empty` should allow `@import` statements

### ISSUE THREAD

**Issue by ghiculescu** [stylelint/stylelint#4109]
> Clearly describe the bug

`block-no-empty` should allow `@import` statements, but it's currently crashing with error "Cannot read property 'every' of undefined"

> Which rule, if any, is the bug related to?

`block-no-empty`

> What CSS is needed to reproduce the bug?

```css
@import "foo.css";
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "block-no-empty": [true, {
      "ignore": ["comments"]
    }]
  }
}
```

> Which version of stylelint are you using?

e.g. `10.1.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

No crash, no lint error.

> What actually happened (e.g. what warnings or errors did you get)?

e.g. "The following warnings were flagged:"

```shell
TypeError: Cannot read property 'every' of undefined
    at check (/node_modules/stylelint/lib/rules/block-no-empty/index.js:50:47)
```
